Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -166,6 +167,29 @@ public class ConnectionPool {
         unlockPoolLock()
     }
     
+    private func increasePoolSize() -> Connection? {
+        defer {
+            unlockPoolLock()
+        }
+        lockPoolLock()
```

Comment:
It doesn't really matter in this instance, but I would suggest flipping these round so that the defer is after the lock.